### ISSUE TITLE
dnsRefreshRate is default to 5s - too short

### ISSUE THREAD

**Issue by linsun** [istio/istio#27329]
This has caused issue for our users where it takes longer than 5s when calling external service and the dns refresh rate is too low.  from my sleep pod sidecar config:

```
    {
     "version_info": "2020-09-14T19:55:14Z/12",
     "cluster": {
      "@type": "type.googleapis.com/envoy.config.cluster.v3.Cluster",
      "name": "outbound|443||x.com",
      "type": "STRICT_DNS",
      "connect_timeout": "301s",
      "circuit_breakers": {
       "thresholds": [
        {
         "max_connections": 4294967295,
         "max_pending_requests": 4294967295,
         "max_requests": 4294967295,
         "max_retries": 4294967295
        }
       ]
      },
      **"dns_refresh_rate": "5s",**
```
I also found out the default is 60s from envoy - I propose we change default to 60s.

https://www.envoyproxy.io/docs/envoy/latest/api-v2/config/common/dynamic_forward_proxy/v2alpha/dns_cache.proto

```

dns_refresh_rate

    (Duration) The DNS refresh rate for currently cached DNS hosts. If not specified defaults to 60s.

    The refresh rate is rounded to the closest millisecond, and must be at least 1ms.
```

@howardjohn @GregHanson @ramaraochavali thoughts?

[ ] Docs
[ ] Installation
[ x] Networking
[ ] Performance and Scalability
[ ] Extensions and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Version (include the output of `istioctl version --remote` and `kubectl version --short` and `helm version` if you used Helm)**
1.7

**Comment by howardjohn** [comment]
How does dns refresh rate impact slow calls?

Also, I thought we are setting respect_dns_ttl? So assuming the dns response sets a TTL we will not ever use 5s?

**Comment by linsun** [comment]
Ok I confirmed on 1.7 env we are setting respect_dns_ttl to true.

Our users are using 1.6.8/9 and found out the IP addresses kept switching when calling the external service with dns_refresh_rate to 5s.  Setting it to 60s via envoy filter solved this prob.

**Comment by howardjohn** [comment]
Its also in mesh config. I think this is resolved as its not relevant in 1.7 really and if it somehow is its configurable via mesh config
